tests/core/test_processor.py

```diff
@@ -901,6 +902,33 @@ def predict_action_probabilities(
     )
 
 
+async def test_action_unlikely_intent_metadata(default_processor: MessageProcessor):
```

Comment:
how about also testing that the metadata from `PolicyPrediction` becomes part of the tracker metadata?

Reply:
I've just updated this test and added one more explicit check if metadata is correct. Let me know what you think!

Reply:
I think the test is great! It would be slightly nicer to test via the public processor interface (e.g. `execute` or `handle_message`) instead of via protectecd functions.